business-registry-dashboard/devops/vaults.env

```diff
@@ -28,3 +28,8 @@ NUXT_REGISTRIES_SEARCH_API_URL="op://API/$APP_ENV/registries-search-api/REGISTRI
 NUXT_REGISTRIES_SEARCH_API_VERSION="op://API/$APP_ENV/registries-search-api/REGISTRIES_SEARCH_API_VERSION"
 NUXT_REGISTRIES_SEARCH_API_KEY="op://API/$APP_ENV/registries-search-api/REGISTRIES_SEARCH_API_KEY"
 NUXT_X_API_KEY="op://web-url/$APP_ENV/business-registry-ui/X_API_KEY"
+
+#google analytics
+NUXT_GTAG_ID="op://google-analytics/$APP_ENV/developer/GTAG"
+NUXT_GTAG_DEBUG="op://google-analytics/$APP_ENV/developer/GTAG_DEBUG"
+NUXT_GTM_ID="op://google-analytics/$APP_ENV/developer/GTM"
```

Comment:
Hey @pwei1018 , can you give me permissions to this vault in 1Password?

Reply:
Should different UIs use different GA ids?

Reply:
no, only 1 GA ids.